src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  let discount = 0;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    discount = SHORT_TERM_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    discount = LONG_TERM_DISCOUNT;
+  } else {
+    discount = 0;
+  }
```

Comment:
The structure of the 'if' conditions can be improved to avoid nesting and to make the code cleaner. Consider checking for the LONG_TERM condition first and returning immediately if it's met, then checking for the SHORT_TERM condition, and finally returning the total cost without any discounts if neither condition is met.